src/core_plugins/kbn_vislib_vis_types/public/controls/point_series/category_axis.js

```diff
@@ -0,0 +1,33 @@
+import _ from 'lodash';
```

Comment:
remove unused _ imports. similar to other files here. jQuery $-imports are generally unused too.